app/src/features/apiClient/screens/environment/components/environmentView/EnvironmentViewManager.tsx

```
@@ -0,0 +1,22 @@
+      isGlobal: false;
+    }
+  | {
+      isGlobal: true;
+    };
+
+export const EnvironmentViewManager: React.FC<EnvironmentViewManagerProps> = (props) => {
+  const environmentId = props.isGlobal ? GLOBAL_ENVIRONMENT_ID : props.envId;
+  const entityType = props.isGlobal ? ApiClientEntityType.GLOBAL_ENVIRONMENT : ApiClientEntityType.ENVIRONMENT;
+  const entity = useBufferedEntity({ id: environmentId, type: entityType });
+  return <EnvironmentView entity={entity} environmentId={environmentId} isGlobal={props.isGlobal} />;
+  return <>Env</>;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)